lib/elixir/lib/calendar.ex

```diff
@@ -194,6 +234,16 @@ defmodule Date do
   end
 
   @doc """
+  Returns the number of days in a week.
+
+  Always returns 7 days in a week irrespective of the
+  calendar in use.
+  """
+  def days_in_a_week() do
```

Comment:
I agree that clarification is needed here. As for information on calendars that have other 'weeks' than 7-day-long ones, see [the Wikipedia article on weeks in other calendars.](https://en.wikipedia.org/wiki/Week#.22Weeks.22_in_other_calendars}. Most of them are not commonly used, but I believe we should consider them.

One of the calendars that uses a 10-day week is the French Revolutionary calendar, which I thought the D&R book covers? (I have not read the book (hopefully I can find it in the university library), but I have read their earlier two papers and their website with secondary content).